udfs/community/test_cases.yaml

```diff
@@ -0,0 +1,62 @@
+int:
```

Comment:
Should this file be split up per function?
Seems like this could get very long as number of UDFs grows.